form.php

```diff
@@ -0,0 +1,39 @@
+<?php
+session_start();
+$users =  array('anton' => '1224', 'grisha' => '5678', 'ivan' => '5432', 'izya' => '1095');
```

Comment:
try to change it with sub arrays like this: 

```php
<?php
array('anton' => array('password' => '1224', 'name' => 'Anton Babenko'), 'grisha' => ('password' => '5678', 'name' => 'Grisha Ilyasov'));
```

make all rest code work with it.